models/stg_recharge__order.sql

```diff
@@ -41,24 +41,24 @@ final as (
         cast(shipped_date as {{ dbt.type_timestamp() }}) as shipped_date,
         address_id,
         address_is_active,
-        billing_address_first_name || ' ' || billing_address_last_name as billing_address_full_name,
-        billing_address_address_1 as billing_address_line_1,
-        billing_address_address_2 as billing_address_line_2,
-        billing_address_city,
-        billing_address_province,
-        billing_address_country,
-        billing_address_zip,
-        billing_address_company,
-        billing_address_phone,
-        shipping_address_first_name || ' ' || shipping_address_last_name as shipping_address_full_name,
-        shipping_address_address_1 as shipping_address_line_1,
-        shipping_address_address_2 as shipping_address_line_2,
-        shipping_address_city,
-        shipping_address_province,
-        shipping_address_country,
-        shipping_address_zip,
-        shipping_address_company,
-        shipping_address_phone,
+        billing_address_first_name || ' ' || billing_address_last_name as billing_address_full_name, 
+        billing_address_address_1 as billing_address_line_1, 
+        billing_address_address_2 as billing_address_line_2, 
+        billing_address_city, 
+        billing_address_province, 
+        billing_address_country, 
+        billing_address_zip, 
+        billing_address_company, 
+        billing_address_phone, 
+        shipping_address_first_name || ' ' || shipping_address_last_name as shipping_address_full_name, 
+        shipping_address_address_1 as shipping_address_line_1, 
+        shipping_address_address_2 as shipping_address_line_2, 
+        shipping_address_city, 
+        shipping_address_province, 
+        shipping_address_country, 
+        shipping_address_zip, 
+        shipping_address_company, 
+        shipping_address_phone, 
```

Comment:
Same comment about these all being custom fields. We should let the variables handle these.

Reply:
Removed.